applications/ShapeOptimizationApplication/python_scripts/analyzer_factory.py

```diff
@@ -56,4 +59,9 @@ def __IsInternalAnalyzerRequired(self, optimization_settings):
                 return True
         return False
 
+    # --------------------------------------------------------------------------
+    def __ResetPossibleShapeModificationsDuringAnalysis( self ):
```

Comment:
__ResetPossibleShapeModifications**From**Analysis

Reply:
The first one is only concerned with reverting any (intended or unintended) shape modifications done during analysis. The other basically corresponds to the "analysis.clear()" function. As the latter is not existing yet, we, for now, cannot do much more than making sure that motion variables and shape modifications are cleared. However, this is the place, where the clear-function should be called in the future.